python/cuda_parallel/pyproject.toml

```
@@ -21,6 +21,8 @@ dependencies = [
   "numba>=0.60.0",
   "cuda-python==12.*",
```

I think we need to pin `>=12.9.0`

---

Perhaps pin it tighter in case we turn path finder into public api in the next patch release? (we can afford to do this before this projects gets picked up by the crowd)
```suggestion
  "cuda-python==12.9.0",
```